snuba/cli/subscriptions_executor.py

```diff
@@ -49,6 +50,10 @@
     default="snuba-subscription-executor",
     help="Consumer group used for consuming the scheduled subscription topic/s.",
 )
+@click.option(
+    "--slice-id",
+    help="The slice to write on/read data from",
```

Comment:
Let's call it "the slice to load scheduled queries from", because the executor is actually capable of executing queries for any slice (and during periods of transition it will execute queries across slices)

Reply:
right, good point